### DISCUSSION TITLE
Labels of Advanced analytics stats in the chart legend can't be changed

### DISCUSSION THREAD

**Discussion by petkoaa** [apache/superset#29645] [Ideas]
Hello,
I've noticed that  the labels that show up in the chart legend for "advanced analytics" can´t be changed but are inferred by percentage_  difference_ etc +  the name of the Metric used + time frame. Since the advanced analytics are predefined already I don't know if a field to change the Name for every Advanced analytic is necessary, but maybe change the prefix that is added to something that makes more sense to have in a chart legend :
![Screenshot_20240719_170230](https://github.com/user-attachments/assets/c4ace2a9-96bd-48a6-86c6-29fb82adaa10)

If I'm wrong and some workaround is possible I do apologize and humbly ask to be pointed in that direction. :pray: 
Thanks in advance ^^

**Comment by PittOpn** [comment]
+1

**Comment by FerreiraPablo** [comment]
+1
